osmaxx/excerptexport/models/output_file.py

```diff
@@ -40,6 +47,11 @@ def __str__(self):
             + ('file: ' + os.path.basename(self.file.name) + ', ' if (self.file and self.file.name) else '') \
             + 'identifier: ' + str(self.public_identifier)
 
+    def get_filename(self):
```

Comment:
As this returns an empty string when `self.file` is `None`, rename it to `get_filename_display` or similar. (I'd expect a method named `get_filename` to return `None` or raise an Error if there is no file.)